contrib/test/integration/system-packages.yml

```diff
@@ -147,3 +147,6 @@
 - name: Update all packages RHEL 8 and Fedora >= 30
   shell: sudo yum update -y
   when: supports_python3 is defined and supports_python3
+
+- name: Allow processes inside of containers to send ping
+  shell: sysctl -w net.ipv4.ping_group_range '0   2147483647'
```

Comment:
We have to similarly give the sysctl to the circle tests. I'm not really sure how to do that. Update the image maybe? @saschagrunert thoughts?